### PR TITLE
Complete Draco-CQL

### PR DESCRIPTION
**Pull request by haldenl**
Adjust weights to match CompassQL optimal recommendations.

Results attached.
[Draco-CQL.pdf](https://github.com/domoritz/draco/files/1854517/Draco-CQL.pdf)

### CODE REVIEW

File: data/compassql_examples/input/3d-NxOxQ.json [hunk 1]
```diff
@@ -5,11 +5,11 @@
     {
       "channel": "?",
       "field": "Cylinders",
-      "type": "nominal"
+      "type": "ordinal"
```
Comment (domoritz) [inline data/compassql_examples/input/3d-NxOxQ.json]: Why change the input?

File: debug.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (domoritz) [inline debug.json]: Plese delete this one

File: notebooks/violations.ipynb [hunk 3]
```diff
@@ -11,12 +11,137 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```
Comment (domoritz) [inline notebooks/violations.ipynb]: Please run the first cell one more time to remove these warnings

File: data/compassql_examples/input/3d-NxOxQ.json (hunk 1 above)
Comment (haldenl) [reply data/compassql_examples/input/3d-NxOxQ.json]: these are all flip-flopped (cylinder should be ordinal, origin should be nominal)

Comment (domoritz) [reply data/compassql_examples/input/3d-NxOxQ.json]: Were they wrong in the original queries?

**Pull request merged**